src/sass/layouts/_advantages.scss

```diff
@@ -1,9 +1,9 @@
 .advantages__list {
     display: flex;
     flex-direction: column;
-    gap: 30px;
+    gap: 27px;
     padding-top: 42px;
-    padding-bottom: 36px;
+    padding-bottom: 60px;
 }
 
 .advantages__amount {
```

Comment:
Please style li instead of deleted gap
Add margin-bottom: 28px; to advantages__item 
Use last :last-child for advantages__item in order to not add margin on last li